Card 22 Jaw-Thrust Airway Translation Upload

farrellgraymond opened the pull request:
As with card 21, name is copy/paste from DM Ukraine. The direct English translation is "removal of the jaw", maybe that should be changed, maybe it makes sense, maybe it's just a bad translation on my end & it makes sense in Ukrainian? Rest of the translations are me, through Google. I will wait a bit for this to be looked over before I merge it. Thanks.

straxus reviewed (changes requested):
Based on review by Kate Bohuslavska, can you please update the title of the 22 UKR card to:

`Маневр висування нижньої щелепи`

farrellgraymond commented:
[22 UKR Jaw-Thrust Airway Maneuver PDF.pdf](https://github.com/farrellgraymond/TCCC-Pixlr/files/12331414/22.UKR.Jaw-Thrust.Airway.Maneuver.PDF.pdf)
![22 UKR Jaw-Thrust Airway Maneuver PNG](https://github.com/farrellgraymond/TCCC-Pixlr/assets/140699253/ee70d5b6-5ff3-4983-9339-d3502a8d4f9a)
Updated Card 22 UKR title as requested. Changed "NOTE" (ПРИМІТКА) to "IMPORTANT" (ВАЖЛИВО) in top line regarding use for spinal trauma injuries. I see it actually translates as "importantly", hope that is fine, I've used it elsewhere in a few cards. Thank you. Will merge if all looks OK.

straxus commented:
Thanks for taking care of this!
